ppr-api/src/ppr_api/models/client_code.py

```diff
@@ -95,6 +95,41 @@ def find_by_head_office_code(cls, head_office_id: str = None):
 
         return party_codes
 
+    @classmethod
+    def find_by_account_number(cls, account_number: int):
```

Comment:
where is this used?

Reply:
It's not...yet. It will be in a coming ticket split from this one.